forge-ai/src/main/java/forge/ai/SpellApiToAi.java

```diff
@@ -56,6 +56,7 @@ public enum SpellApiToAi {
             .put(ApiType.Clash, ClashAi.class)
             .put(ApiType.ClassLevelUp, AlwaysPlayAi.class)
             .put(ApiType.Cleanup, AlwaysPlayAi.class)
+            .put(ApiType.Cloak, CloakAi.class)
```

Comment:
I wonder if it should just be mapped to Manifest since the decisions are so closely related 🤔

Reply:
they are slightly different so they might need extra logic later